src/api/firebase.js

```diff
@@ -15,6 +15,16 @@ export function streamListItems(listId, handleSuccess) {
 	return onSnapshot(listCollectionRef, handleSuccess);
 }
 
+/**
+ * Check existence of list in Firestore associated with user token input.
+ * @param {string} userTokenInput The user's token input requesting to be validated
+ */
+export async function validateToken(userTokenInput) {
+	const listCollectionRef = collection(db, userTokenInput);
+	const listSnapshot = await getDocs(listCollectionRef);
```

Comment:
Curious how line 23 and 24 work together? We looked up the firebase "getDocs" function. Still a little fuzzy on how it all works.

Reply:
Hey Amy! If I'm understanding things correctly, accessing data from a firestore db is a two-part process. You first get a reference to a collection in the db using the `collection` method. I like to think of this like requesting an address. And with that reference, you're then able to actually query the db using `getDocs` to take "snapshots" of the db state at a particular instance in time. I like to think of this like a camera taking a picture of the db and capturing its state at an instance in time. 

Still hazy with all of this myself though so I'm not sure this is completely right.

Reply:
That's a great explanation!

Reply:
great explanation- thank you @jongranados !